inttest/terraform/test-cluster/output.tf

```diff
@@ -5,3 +5,8 @@ output "controller_external_ip" {
 output "worker_external_ip" {
   value = aws_instance.cluster-workers.*.public_ip
 }
+
+
+output "controller_pem" {
+  value = local_file.aws_private_pem
```

Comment:
I needed this value to setup connection for file and remote provisioners when using it in the module.